crates/infra/cli/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+pub mod commands;
```

Comment:
To give more context, it's common in Rust to have a CLI have its core functionality as a lib that has a wrapper around it for a binary. This mainly streamlines sharing the entry point between the build.rs and the binary, without having to `#[path]` rename a module, but in theory also would allow to re-use some bits of logic, if needed.